src/swap/SwapScreen.tsx

```diff
@@ -210,6 +210,16 @@ export function SwapScreen({ route }: Props) {
   const handleReview = () => {
     ValoraAnalytics.track(SwapEvents.swap_screen_review_swap)
 
+    const userInput = {
+      toToken: toToken!.address,
+      fromToken: fromToken!.address,
```

Comment:
is there a way we could avoid the non-null assertion operators here? Even if it's just throwing a descriptive error message if either is null, instead of a type error, to make our lives a bit easier if we have to debug it?

Reply:
yep! 👍

Reply:
i think Jacob has work in flight to update this screen to use tokenId which will fix this problem, so i don't think it's worth doing here

Reply:
oh yes that's right, it's why I didn't touch it either.